src/python/pants/core/goals/export.py

```diff
@@ -129,6 +154,12 @@ class ExportSubsystem(GoalSubsystem):
         "e.g., Python resolves are exported as virtualenvs.",
     )
 
+    binaries = StrListOption(
+        flag_name="--bin",  # `bin` is a python builtin
+        default=[],
+        help="Export the specified binaries.",
```

Comment:
Potentially silly question, but are names to use here decided by the `options_scope`? If so, can we remind users of that here? E.g.

```suggestion
        help="Export the specified binaries. To select a binary, provide its subsystem scope name, as used for setting its options.",
```